python-server/test_server.py

```diff
@@ -10,10 +10,10 @@
 import re
 
 skydentity_creds = '../tokens/.cloud_creds/skydentity-token.json'
-
+skydentity_creds = "/Users/samyu/.cloud_creds/gcp/proxy/sky-identity-ac2febc1b9b3.json"
```

Comment:
Should maybe remove this from the commit?

Reply:
Yes, updated